File: yurt-jdbc/src/test/java/com/github/liulus/yurt/model/GoodsQuery.java
package com.github.liulus.yurt.model;

import com.github.liulus.yurt.convention.data.PageQuery;

import java.time.LocalDateTime;
import java.util.List;

/**
 * @author liulu
 * @version V1.0
 * @since 2020/11/10
 */
public class GoodsQuery extends PageQuery {

    private String code;
    private List<String> codes;
    private String fullName;
    private Integer inventory;
    private Double price;
    private LocalDateTime startTime;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public List<String> getCodes() {
        return codes;
    }

    public void setCodes(List<String> codes) {
        this.codes = codes;
    }

    public String getFullName() {
        return fullName;
    }

    public void setFullName(String fullName) {
        this.fullName = fullName;
    }

    public Integer getInventory() {
        return inventory;
    }

    public void setInventory(Integer inventory) {
        this.inventory = inventory;
    }

    public Double getPrice() {
        return price;
    }

    public void setPrice(Double price) {
        this.price = price;
    }

    public LocalDateTime getStartTime() {
        return startTime;
    }

    public void setStartTime(LocalDateTime startTime) {
        this.startTime = startTime;
    }
}

File: yurt-data/yurt-data-jdbc/src/test/kotlin/com/github/liulus/yurt/model/GoodsQuery.kt
package com.github.liulus.yurt.model

import com.github.liulus.yurt.convention.data.PageQuery
import java.time.LocalDateTime

/**
 *
 * @author liulu
 * @version V1.0
 * @since 2020/11/6
 */
class GoodsQuery : PageQuery() {

    var fullName: String? = null
    var code: String? = null
    var inventory: Int? = null
    var price: Double? = null
    var startTime: LocalDateTime? = null
}